File: Lista de Exercícios/Python/3. Encapsulamento/Encapsulamento.py
class ContaBancaria:
    def __init__(self, titular):
        self.titular = titular
        self.__saldo = 0

    def depositar(self, valor):
         if valor > 0:
              self.__saldo += valor
    
    def sacar(self, valor):
         if 0 < valor <= self.__saldo:
              self.__saldo -= valor

File: Lista de Exercícios/Go/3. Encapsulamento/Encapsulamento.go
package main

import (
	"fmt"
)

type ContaBancaria struct {
	titular string
	saldo   float
}

func (c *ContaBancaria) depositar(valor float) {
	if valor > 0 {
		c.saldo += valor
	}
}

func (c *ContaBancaria) sacar(valor float) {
	if 0 < valor && valor <= c.saldo {
		c.saldo -= valor
	}
}

func (c ContaBancaria) exibirSaldo() {
	fmt.Printf("Saldo atual: %.2f\n", c.saldo)
}

func main() {
	
}